packages/next-auth-sample/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@logto/next-auth-sample",
+  "version": "2.3.13",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "next": "^15.2.3",
+    "next-auth": "5.0.0-beta.27",
```

Comment:
just wanted to double check if it has a new stable version

Reply:
No, this is the official recommended version, and only v5 has native Logto provider.